src/rust/plugin-registry/tests/test_deploy_plugin.rs

```diff
@@ -246,7 +265,17 @@ async fn test_teardown_plugin() {
         .expect("failed to get plugin deployment")
         .plugin_deployment();
 
-    assert_eq!(plugin_deployment.plugin_id(), plugin_id);
-    assert!(!plugin_deployment.deployed());
-    assert_eq!(plugin_deployment.status(), PluginDeploymentStatus::Success);
+    eyre::ensure!(plugin_deployment.plugin_id() == plugin_id, "plugins equal");
+    eyre::ensure!(plugin_deployment.deployed(), "plugin deployed");
+    eyre::ensure!(
+        plugin_deployment.status() == PluginDeploymentStatus::Success,
+        "deployment successful"
+    );
```

Comment:
the above eyre-ification came about because of the change to `-> eyre::Result` on line 211. I was like oh i guess i should remove these asserts